packages/components/buttongroup/src/ButtonGroup.styles.ts

```diff
@@ -0,0 +1,76 @@
+import { css } from 'emotion';
+import tokens from '@contentful/f36-tokens';
+import type { ButtonGroupVariants } from './types';
+import { CSSObject } from '@storybook/theming';
+
+const getGroupContentStyle = (
+  variant: ButtonGroupVariants,
+  withDivider: boolean,
+) => {
+  if (variant === 'separate') {
+    return {
+      marginLeft: tokens.spacingS,
```

Comment:
is it always `spacingS` according to design guidelines?

Reply:
There's no example of a separates button group on figma, I added `spacingS` to have the variant, but would need to double check with the designers.

Reply:
yeah, let's double-check that

Reply:
I may remove the option of having them as separate for now and we add it back later on if we actually need it, wdyt?

Reply:
i'd rather leave it as `spacingS` and then add a new property to control it

Reply:
Makes sense, I'll add another prop to control the spacing, and keep the spacingS as default and move it to the Button component then. 😄